backend/inspirehep/utils.py

```diff
@@ -85,6 +85,21 @@ def get_failure_message_for_index_record(task_name, exception, **kwargs):
     return format_failure_message_for_single(task_name, exception, kwargs.get("uuid"))
 
 
+def get_failure_message_for_disambiguate_records(task_name, exception, *args, **kwargs):
+    if isinstance(args[0], list):
+        return format_failure_message_for_multiple(task_name, exception, args[0])
+    return format_failure_message_for_single(task_name, exception, args[0])
+
+
+def get_failure_message_for_celery_starmap(task_name, exception, *args, **kwargs):
+    task_name = kwargs.get("task")["task"]
+    uuids = []
+    for arg in kwargs.get("it"):
+        uuids.append(arg[0])
```

Comment:
You can make it as a comprehension `uuids = [arg[0] for arg in kwargs.get("it", [])]`